### PR TITLE
add DevTools exercise for CSS 'visible' tag 

### PR DESCRIPTION
**Pull request by lizraeli**

### CODE REVIEW

File: unit1-intro-to-programming-in-javascript/chrome-dev-tools/visible/visible.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- difficulty: 
+- format: 
+
+### Question
+For this exercise, you will need to run the relevant html page on a Chrome browser. After entering an input, open `Chrome Developer Tools` on the tab `elements`.
+
+1. There a hidden "Add" button.  Un-hide it and add some items.
+2. Add a some new elements to the list by going to the tab `Resources`  **->** `Local Storage`. You may need to refresh the page.
+
+Advanced CSS:
```
Comment (bensussman) [inline unit1-intro-to-programming-in-javascript/chrome-dev-tools/visible/visible.md]: Should there be more down here?

**Comment by bensussman** [comment]
Does this belong in "chrome dev tools" or in the "HTML & CSS" section (which has Chrome Dev Tools as one of the sub-topics)?

The top-level Chrome Dev Tools topic was designed for specifically javascript debugging (like actual live code debugging). We have 3 days dedicated to HTML & CSS, and we should definitely teach them CDT then.

**Comment by lizraeli** [comment]
@bensussman

File: unit1-intro-to-programming-in-javascript/chrome-dev-tools/change-variable-value/assets/rock-paper-scissors.js~ [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    } else {
+      winner = TIE;
+    }
+
+    return winner;
+  }
+
+
+
+}
```
Comment (bensussman) [inline unit1-intro-to-programming-in-javascript/chrome-dev-tools/change-variable-value/assets/rock-paper-scissors.js~]: is the filename appended with `~` unnecessarily?

**Pull request merged**